commit 818acb37bfb5ebc3ac6d6979c183b13411211bc9 (refs/changes/91/1391/1)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2017-02-01T23:57:06+08:00 (2 years, 8 months ago)
    
    PUBAPI-1360: Test failing due to improved error message from fwapi

diff --git a/test/machines/firewall-rules.js b/test/machines/firewall-rules.js
index 85f03d6..90f0719 100644
--- a/test/machines/firewall-rules.js
+++ b/test/machines/firewall-rules.js
@@ -122,7 +122,12 @@ module.exports = function (suite, client, other, machine, callback) {
             t.deepEqual(body, {
                 code: 'InvalidParameters',
                 message: 'Invalid parameters',
-                errors: []
+                errors: [ {
+                    code: 'InvalidParameter',
+                    field: 'rule',
+                    message: 'Subnet "10.99.99.0/23" is invalid (bits set to ' +
+                        'right of mask)'
+                } ]
             });
 
             t.end();
